WordPress/Classes/ViewRelated/Blog/Site Picker/SiteSwitcherView.swift

```diff
@@ -79,9 +81,13 @@ private struct SiteSwitcherView: View {
             if !isSearching {
                 HStack {
                     Spacer()
-                    FAB(action: addSiteAction)
-                        .padding(.trailing, 20)
-                        .padding(.bottom, UIDevice.current.userInterfaceIdiom == .pad ? 20 : 0)
+                    ZStack {
+                        PopoverSourceView(view: popoverSourceView)
+                        FAB(action: { addSiteAction(popoverSourceView) })
+                    }
```

Comment:
Out of curiosity, any reason not using SwiftUI [`confirmationDialog`](https://developer.apple.com/documentation/swiftui/view/confirmationdialog(_:ispresented:titlevisibility:presenting:actions:message:)-8y541) to implement this action sheet?

Reply:
Sorry, I see the PR description now. You have tried that.